search/search_repository_blackbox_test.go

```diff
@@ -146,6 +147,149 @@ func (s *searchRepositoryBlackboxTest) TestSearchWithJoin() {
 	})
 }
 
+func (s *searchRepositoryBlackboxTest) TestSearchBoardColumnID() {
+	s.T().Run("boardcolumn", func(t *testing.T) {
+		fxt := tf.NewTestFixture(t, s.DB,
+			tf.CreateWorkItemEnvironment(),
+			tf.WorkItemBoards(2),
+			tf.WorkItems(3, func(fxt *tf.TestFixture, idx int) error {
+				switch idx {
+				case 0:
+					fxt.WorkItems[idx].Fields[workitem.SystemBoardcolumns] = [2]string{
+						fxt.WorkItemBoards[0].Columns[0].ID.String(),
+						fxt.WorkItemBoards[1].Columns[1].ID.String(),
```

Comment:
If the test fixture has created `one` workitem type, how can there be a `fxt.WorkItemBoards[1].Columns[1]` ? Each type represents one column IIUC.

Reply:
There are two boards created in the fixture setup. Each one creates a type group in turn as a dependency. And every board has the columns that are defined in the respective template, so I think this is correct. The types are not directly connected to the columns.

Reply:
Thanks @michaelkleinhenz